Source/WebCore/Modules/WebGPU/Implementation/WebGPUDeviceImpl.cpp

```diff
@@ -387,8 +387,8 @@ static auto convertToBacking(const ComputePipelineDescriptor& descriptor, Conver
 
 RefPtr<ComputePipeline> DeviceImpl::createComputePipeline(const ComputePipelineDescriptor& descriptor)
 {
-    return convertToBacking(descriptor, m_convertToBackingContext, [backing = m_backing, &convertToBackingContext = m_convertToBackingContext.get()](const WGPUComputePipelineDescriptor& backingDescriptor) {
-        return ComputePipelineImpl::create(adoptWebGPU(wgpuDeviceCreateComputePipeline(backing.get(), &backingDescriptor)), convertToBackingContext);
+    return convertToBacking(descriptor, m_convertToBackingContext, [backing = m_backing, convertToBackingContext = m_convertToBackingContext](const WGPUComputePipelineDescriptor& backingDescriptor) {
```

Comment:
Can we mark the `convertToBacking()` lambda as `NOESCAPE const T&` instead?

Reply:
I couldn't figure this one out. I left this unchanged for now.